cmd/src/setup.rs

```diff
@@ -220,6 +221,15 @@ pub fn initial_logger(config: &TiKvConfig) {
             });
         }
     };
+
+    // Set redact_info_log.
+    let redact_info_log = if let Some(redact_info_log) = config.security.redact_info_log {
+        redact_info_log
+    } else {
+        config.security.encryption.data_encryption_method != EncryptionMethod::Plaintext
```

Comment:
`redact_info_log` should be independent of encryption setting. Can you help me update this?

Reply:
Now it is independent from security config.